### PR TITLE
style(#42): 🎨 Change default body font for Signika

### LINKED ISSUE #42: Change the Typeface
Current Typefaces was selected quickly to not block prototyping. We discussed with PeL! C's and Com teams for more fitting fonts.

# Title : GROBOLD
Grobold was selected because it's a licensed font we bought the right to use. It fit with the existing PeL! design language.
 
image

# Body : Signika
Signika was picked by the Com team based on comparative renders with others fonts. Signika is compact but really easy to read, sans serif, have multiple weights so it allow us flexible styles, and have some curves that blend well with FO! rounded corners design language.

image

---

image

### PR DESCRIPTION
**Pull request by SimonPistache**
closes #42 .

### CODE REVIEW

**Comment by SimonPistache** [comment]
Need some testing to see how the font render (size, weight) ...

File: .vscode/settings.json
Comment (chatondearu) [inline .vscode/settings.json:1]: I think this changes were automatically generated and are not a voluntary addition. we could remove it ?

**Review by chatondearu — CHANGES_REQUESTED** [review]
You need to add the google font also in the Nuxt layer settings
### here:
https://github.com/Paris-est-Ludique/ForceOrange/blob/737fd5b7deee447c6990cd29b6e4d3c30907cdeb/modules/shared/nuxt.config.ts#L56

File: .vscode/settings.json
Comment (SimonPistache) [reply .vscode/settings.json:1]: Yes, it's a VScode addon that manage Conventional Commit and help set commits compatible with that formatting.

Let me remove that.

**Review by chatondearu — APPROVED** [review]
LGTM

**Pull request merged**